lnd_test.go

```diff
@@ -3537,7 +4071,7 @@ func testAsyncPayments(net *networkHarness, t *harnessTest) {
 
 	const (
 		timeout    = time.Duration(time.Second * 5)
-		paymentAmt = 100
+		paymentAmt = 1000
```

Comment:
had to bump the base payment amt, otherwise the total amount being swept results in an insufficient fee failure on publication. i believe this could be reverted after properly handling publications errors